### ISSUE TITLE
getGuestQuoteId issue with ScandiPWA 5.3.3

### ISSUE THREAD

**Issue by Ylmzef** [mollie/magento2-scandipwa#6]
I'm getting following issue after upgrading to the latest version of ScandiPWA 5.3.3

```
Failed to compile.

./packages/@mollie/scandipwa/src/plugin/Checkout.container.plugin.js
Attempted import error: 'getGuestQuoteId' is not exported from '/var/www/vhosts/xxx.xxx/httpdocs/pwa/xxx/src/util/Cart'.
```

**Comment by Ylmzef** [comment]
Anohter issue:

```
Failed to compile.

./packages/@mollie/scandipwa/src/component/MollieIssuer/MollieIssuer.container.js
Attempted import error: 'getGuestQuoteId' is not exported from '/var/www/vhosts/xxx.xxx/httpdocs/pwa/xxx/src/util/Cart'.
```

**Comment by Ylmzef** [comment]
Any update please?

**Comment by Frank-Magmodules** [comment]
Hi @Ylmzef ,  
We ran into quite some issues with the latest version of ScandiPWA which causes some blocks in testing our solution but it seems a simple downgrade fixed that. We are currently testing our fixes and will wrap this into a new release beginning next week. Thank you for your patience!

**Comment by Marvin-Magmodules** [comment]
Hi @Ylmzef,

We are happy to share that we've just released the new [1.0.2](https://github.com/mollie/magento2-scandipwa/releases/tag/1.0.2) version where we have fixed this issue. Thank you for opening this issue and for the detailed issue report. We are closing this issue now but please feel free to reopen the issue if this still occurs.
